src/components/chip/index.tsx

```diff
@@ -76,7 +76,7 @@ export type ChipProps = ViewProps & TouchableOpacityProps & {
   /**
    * Additional icon props
    */
-  iconProps?: ImageProps
+  iconProps?: Partial<ImageProps>;
```

Comment:
1. This means that **any** prop from `ImageProps` is optional, although I'm not sure there is any other prop that is mandatory, there might be in the future, so maybe another solution is better - look in `CardImage`.

2. There's still an error in the private lib with this change.

Reply:
Actually that was my second option, I thought the one I chose is more generic..

Reply:
I don't think we want generic here, we want it specific.
Not that I like the option I suggested (not very pretty), but I think it's better than this one.